demo.js

```diff
@@ -13,7 +13,17 @@
 
                     onChange: function (model) {
                         this.setState({model: model});
-                    }.bind(this)
+                    }.bind(this),
+                    lang: 'ru',
+                    messages: {
+                        period: {
+                            singleDay: 'Один день',
+                            oneNight: '1 ночь',
+                            xNights: '{count} ночей'
+                        },
+                        checkInDay: 'День заселения',
+                        checkOutDay: 'День выселения'
+                    }
```

Comment:
Please keep the demo texts in English. Otherwise, it's hard to grasp the meaning for international users looking at https://ikr.su/h/react-period-of-stay-input/demo.html linked from the README. OR! Let's do the 2 control instances on the demo page: one in English -- with the defaults, and one in Russian.